TicTacToe/test/tictactoe/TicTacToeTest.java

```diff
@@ -1,36 +1,96 @@
 package tictactoe;
 
 import org.junit.Test;
-import static org.junit.Assert.*;
-
-public class TicTacToeTest {
+import org.junit.Assert;
 
+public class TicTacToeTest
+{
     @Test
     public void testIsWinner() {
+        final TicTacToe test1 = new TicTacToe();
+        for (int i = 0; i < 3; ++i) {
+            test1.playMove('X', 0, i);
+        }
+        Assert.assertEquals((Object)test1.isWinner('X'), (Object)true);
+        final TicTacToe test2 = new TicTacToe();
+        for (int j = 0; j < 3; ++j) {
+            test2.playMove('X', j, 0);
+        }
+        Assert.assertEquals((Object)test2.isWinner('X'), (Object)true);
+        final TicTacToe test3 = new TicTacToe();
+        for (int k = 0; k < 3; ++k) {
+            test3.playMove('O', k, k);
+        }
+        Assert.assertEquals((Object)test3.isWinner('O'), (Object)true);
+        final TicTacToe test4 = new TicTacToe();
+        Assert.assertEquals((Object)test4.isWinner('X'), (Object)false);
```

Comment:
Your only false checking is for an empty board.  What about boards that are partially full?  When X wins, you should also verify that O does NOT win.